Issues/Week205.md

```diff
@@ -1,14 +1,14 @@
 **Articles**
 
-* [MVC.](http://codeplease.io/2017/11/19/mvc/), by [@peres](https://twitter.com/peres)
+* [MVC](http://codeplease.io/2017/11/19/mvc/), by [@peres](https://twitter.com/peres)
```

Comment:
Looks like the original title has the dot ([http://codeplease.io/2017/11/19/mvc/](http://codeplease.io/2017/11/19/mvc/)), so I think we should keep it

Reply:
Yep. I looked at it. :)

Reply:
I deleted it because it looked very strange... Like ".," and it has no sense. 
That is why I decided to delete it. Sorry, it was a mistake.